### ISSUE TITLE
Reference documentation page is empty

### ISSUE THREAD

**Issue by ovatsus** [fsprojects/FSharp.Control.AsyncSeq#85]
http://fsprojects.github.io/FSharp.Control.AsyncSeq/reference/index.html

**Comment by eulerfx** [comment]
Thanks, I'll take a look.

**Comment by wilsoncg** [comment]
I tried a local build but the GenerateHelp build target is failing with the following error:
```powershell
Starting Target: GenerateHelp (==> CleanDocs) 
Deleting D:\Projects\fsharp\FSharp.Control.AsyncSeq\docs\content\release-notes.md               
Deleting D:\Projects\fsharp\FSharp.Control.AsyncSeq\docs\content\license.md               
FSharp.Formatting Information: 0 : FSharp.Formatting Logging setup!
Yaaf.FSharp.Scriping Information: 0 : Yaaf.FSharp.Scripting Logging setup!
System.MissingMethodException: Method not found: 'Microsoft.FSharp.Core.FSharpFunc`2<Boolean,Microsoft.FSharp.Collections.FSharpList`1<System.String>> Fake.FileHelper.CopyRecursive(System.String,System.String)'.
   at FSI_0003.copyFiles()
   at <StartupCode$FSI_0003>.$FSI_0003.main@() in D:\Projects\fsharp\FSharp.Control.AsyncSeq\docs\tools\generate.fsx:line 123
Stopped due to error
```

After more investigation, it looks like there is a mismatch between the versions of FSharpCore.dll. FakeLib v4.64.13.0 is referencing FSharp.Core v4.4.1.0 which is used as part of FSharp.Formatting in generate.fsx, while the main project build is referencing FSharp.Core v3.1.2 (net45) & FSharp.Core v4.3.2 (netstandard2.0). 

Further digging reveals Fakelib.dll is now legacy [https://github.com/fsharp/FAKE/issues/2026](https://github.com/fsharp/FAKE/issues/2026).

More work would be needed to switch the build to use FAKE v5.0, and/or use the updated scaffold project. I'm not sure which of these is the best option, or if this is necessary right now.

How are the API reference html pages uploaded to the github pages site after they are generated locally? I can see a ReleaseDocs target in the build, is that run by a project maintainer?

**Comment by wilsoncg** [comment]
@eulerfx /@dsyme what's the steps for updating

**Comment by wilsoncg** [comment]
Resolved in #103
